Cargo.toml

```diff
@@ -26,3 +26,6 @@ mockstream = "0.0.3"
 [features]
 testing = ["parsec-interface/testing", "no-fs-permission-check"]
 no-fs-permission-check = []
+
+[patch.crates-io]
+parsec-interface = { git = "https://github.com/parallaxsecond/parsec-interface-rs.git" }
```

Comment:
Is it possible to scope this down to a particular commit? Otherwise we could end up making breaking changes.

Reply:
Done.